src/hbbft_acs.erl

```diff
@@ -269,7 +268,12 @@ deserialize(#acs_serialized_data{done=Done, n=N, f=F, j=J, rbc=RBCMap, bba=BBAMa
 deserialize(M, SK) ->
     {RBCMap, M1} = maps:take(rbc, M),
     {BBAMap, M2} = maps:take(bba, M1),
-    M3 = maps:map(fun(_K, V) -> binary_to_term(V) end, M2),
+    M3 = maps:map(fun(_K, V) when is_binary(V) ->
```

Comment:
Is maps:filter clearer? Can also use a function head match instead of a guard for `is_binary`:```
fun(_K, <<V/binary>>) -> ...
```

Reply:
I thnk this code is unnecessary entirely, will remove.